lrauv_ignition_plugins/src/ScienceSensorsSystem.cc

```diff
@@ -713,25 +1009,41 @@ void ScienceSensorsSystem::PostUpdate(const ignition::gazebo::UpdateInfo &_info,
         sensorPosENU.Y(),
         sensorPosENU.Z());
 
-    // kNN search (alternatives are voxel search and radius search. kNN
-    // search is good for variable resolution when the distance to the next
-    // neighbor is unknown).
-    int k = 8;
+    // Visualization never shows up when interpolation is always called.
+    // Quick fix: Don't need to interpolate EVERY PostUpdate(). That's overkill.
```

Comment:
```suggestion
    // Quick fix: Don't need to interpolate EVERY PostUpdate(). That's overkill.
    // TODO(tfoote) update this threshold to be data dependent not hard coded with INTERPOLATE_DIST_THRESH
```

Reply:
Fixed in #110 and now merged into this PR in 0af6a57